src/config.ts

```diff
@@ -107,6 +143,20 @@ export const rollupClient = createPublicClient({
   transport: http(),
 }).extend(publicActionsL2()) as PublicL2ClientWithChain;
 
+export const L1ToL2Mes = getContract({
+  address: import.meta.env.VITE_L1_TO_L2_MES_ADDRESS,
+  abi: l1tol2mesAbi,
+  client: rollupClient,
+});
```

Comment:
What chain is it that you're attempting to support with this L1ToL2Messenger contract? This repository is intentionally tightly coupled with OP Stack and that's not a contract that's used for it.